src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-network-request-settings.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Configure mobile network request settings
+tags:
+  - Mobile monitoring settings
+  - Network requests
+metaDescription: Block or allow hostnames, create rules for status codes, and create aliases for hostnames.
+---
+
+import mobileNewStatusRule from 'images/mobile_screenshot-crop_new-status-rule.png'
+
+import mobileCreateAlias from 'images/mobile_screenshot-crop_create-alias.png'
+
+import mobileAllowHostname from 'images/mobile_screenshot-crop_allow-hostname.png'
+
+The mobile network request settings page can help you customize how network metrics and events are reported. You can [block or allow hostnames](#blockallow-hostnames), [ignore HTTP status codes from certain hosts](#ignore-status-codes), and [create aliases to rename hosts or group subdomains](#group-into-alias).
+
+To configure these settings, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile (select a mobile app) > Settings > Network settings**.
+
+## Block/allow hostnames [#blockallow-hostnames]
+
+You can allow or block certain hostnames from being monitored. For example, if you want to only allow subdomains of `ecommerce.com`:
+
+1. On the **Network settings page**, select the **Show only hostnames** tab.
+2. Click **Add a hostname**. 
+3. Enter the hostname `*.ecommerce.com` and click **Add to list**.
+<img
+  title="Allow a hostname"
+  alt="Allow a hostname"
+  src={mobileAllowHostname}
+    />
+Wait up until 2 hours for the rule to be applied.
+
+## Ignore HTTP status codes [#ignore-status-codes]
+
+You can ignore specific HTTP error status code on a hostname. For example, if you want to ignore `401`, `403`, and `404` errors from all `ecommerce.com` subdomains:
+
+1. On the **Network settings page**, select the **Status codes** tab.
+2. Click **Create a status code rule**. 
+3. Enter the host `*.ecommerce.com`.
+4. Enter status codes `401`, `403`, and `404`  and click **Add to list**. 
+<img
+  title="Create a status code rule"
+  alt="Create a status code rule in the UI"
+  src={mobileNewStatusRule}
+  />
+
+Wait up until 2 hours for the rule to be applied.
+
+## Create aliases for hostnames [#group-into-alias]
+
+You can create aliases to do things like rename host names that make the most sense to you. For example, if you want to create an alias that groups all subdomains of `ecommerce.com`:
```

Comment:
```suggestion
You can create aliases to rename host names that make the most sense to you. For example, if you want to create an alias that groups all subdomains of `ecommerce.com`:
```

If you can be more specific about what "things like" refers to, then I'd include this preface to the examples. Is it mostly just customizing names, or is it a categorization feature? Otherwise I'd just jump right into what the things are